README.uk-UA.md

```diff
@@ -0,0 +1,211 @@
+# @ekreative/react-native-braintree
+
+## Починаємо
+
+## Спеціально для Android
+Додайте це до свого `build.gradle`
+
+```groovy
+allprojects {
+    repositories {
+        maven {
+            // Braintree 3D Secure
+            // https://developer.paypal.com/braintree/docs/guides/3d-secure/client-side/android/v4#generate-a-client-token
+            url "https://cardinalcommerceprod.jfrog.io/artifactory/android"
+            credentials {
+                username 'braintree_team_sdk'
+                password 'AKCp8jQcoDy2hxSWhDAUQKXLDPDx6NYRkqrgFLRc3qDrayg6rrCbJpsKKyMwaykVL8FWusJpp'
+            }
+        }
+    }
+}
+```
+
+У вашому `AndroidManifest.xml`, `android:allowBackup="false"` можна замінити `android:allowBackup="true"`, він відповідає за резервне копіювання програми.
+
+Крім того, додайте цей фільтр намірів до основної діяльності в `AndroidManifest.xml`
```

Comment:
In the original text, it says:
Also, add this **intent-filter** to your main **activity** in `AndroidManifest.xml`
I would better translate it to:
Крім того, додайте **intent-filter** до основного **activity** в `AndroidManifest.xml`
Cause otherwise this line is a bit misleading.